### ISSUE TITLE
How can I return Object instead of JSON String from Squiggly API

### ISSUE THREAD

**Issue by DheerajKN** [bohnman/squiggly-java#56]
Instead of returning a Simple JSON String, using `Squiggly.stringify` how can I return an Java Object after `ObjectMapper objectMapper = Squiggly.init(new ObjectMapper(), filter);` as been performed on the filter variables.

**Comment by bohnman** [comment]
You can convert your object to a Map:

```java
Map converted = objectMapper.convertValue(someObject, Map.class);
```

or a list of maps

```java
CollectionType collectionType = objectMapper.getTypeFactory().constructCollectionType(List.class, Map.class);
        List<Map> converted = objectMapper.convertValue(someList, collectionType);
```

However, I'd like to learn more about your use case.  Using Squiggly.init with a static filter is usually only useful for demonstration purposes.  Most people are using Squiggly in a web application.

**Comment by DheerajKN** [comment]
After working with the given solution I have found that ObjectMapper inherently provides with convertValue() and readValue() functions that will take JSON String and .class value that will convert it into POJO.
`ObjectMapper objectMapper = Squiggly.init(new ObjectMapper(), filter);
User withRequiredValues = objectMapper.readValue(SquigglyUtils.stringify(objectMapper, userObject), User.class);`

**Comment by DheerajKN** [comment]
But I was asking for some SquigglyUtils Method that can convert give POJO instead of converting it into String using stringify and then converting it into POJO using readValue().
Something like:
```
ObjectMapper objectMapper = Squiggly.init(new ObjectMapper(), filter);
User withRequiredValues = SquigglyUtils.objectify(objectMapper, userObject, User.class);
```

**Comment by DheerajKN** [comment]
Now it's possible using `SquigglyUtils.objectify` method that accepts `ObjectMapper, someObject, someObject.class`.

**Comment by bohnman** [comment]
@DheerajKN I have released version 1.3.15, which includes the new objectify methods.

**Comment by DheerajKN** [comment]
Thanks @bohnman
